PizzaOrder/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css" />
+    <title>Document</title>
+</head>
+<body>
+    <p>Pizza Order</p>
+    <form action="bio.html">
+        <label for="name">Your name:</label>
+        <input type="text" name="name" minlength="3" maxlength="10" value="Michael Scott" required />
+
```

Comment:
Ca sa putem lega un label de un input, trebuie sa folosim atributele for si id:

```suggestion
        <label for="name">Your name:</label>
        <input type="text" id="name" name="name" minlength="3" maxlength="10" value="Michael Scott" required />

```